### ISSUE TITLE
[Bug] Theme options are not display and saved correctly

### ISSUE THREAD

**Issue by SpaceEnergy** [shdwmtr/millennium#148]
### Before Reporting

- [X] I found no existing issues matching my bug
- [X] My issue is not caused by a theme

### Describe the Bug

There is the problem that the theme settings are not saved correctly / are not recognized correctly. When you go into the settings, an option is off according to the checkbox, but is visibly on, but to really turn it off you have to turn the option on, off, on, off (not always restart but in one go).

###### In the picture you can see how the option is off according to the settings, but is visibly on
![{6AC3E70C-9A6C-4A52-B1FA-61D4AEB69FE7}](https://github.com/user-attachments/assets/d2b10d65-4550-472c-82e8-529ee9442ab2)

### Expected Behavior

Correct display of options and saving

### Steps To Reproduce

1. Enable Theme with settings
2. Open settings
3. Enable a option and restart (maybe restart steam several times)
4. Open theme settings
5. And check the option status

### Operating System

Windows

**Comment by shdwmtr** [comment]
I can't seem to reproduce the issue on my end with fluenty or other themes, I can try to reproduce it with your working copy if you'd like.

**Comment by SpaceEnergy** [comment]
sure, i send you a DM with the theme files

**Comment by shdwmtr** [comment]
It doesn't seem anyone else has this issue, so Im genuinely confused as to why you have it. I'm going to keep this open but I have no idea why its happening as it seems to be an issue with Steams components.

**Comment by SpaceEnergy** [comment]
![image](https://github.com/user-attachments/assets/6089bfe9-e7b1-4ea8-95e7-8f41cb8f5f73)

**Comment by xVotexX** [comment]
This is really weird. I'm using SpaceTheme [v1.3.3a] aswell. For me it saves the settings of checked boxes when closing the window and restarting steam via the popup. But the checkboxes or the pages in general are quite buggy. 

1. For some reason, all of the Checkboxes get unchecked if you go to the 'Colors' Tab. 
2. When Checking a Checkbox on one page, the checkbox at the same position on another page gets checked/unchecked aswell.
3. If you check a box on one page then go to another page and there is no box at the same position, once returning to the first page, the checkbox will be unchecked

![steamwebhelper_WtuoFWZgDJ](https://github.com/user-attachments/assets/3932a2ac-783f-41e6-8b7d-7be778c8d77c)

**Comment by yurisuika** [comment]
Ironically, I found out about the ability to make custom settings tabs earlier today from this issue but didn't actually take note of what the issue was concerning. I added custom tabs to my skin and I have now noticed this occurring, but I can't say I recall it happening back when just using the default tab for everything.

Perhaps it does occur there and I never noticed it, but the immediate obviousness of how frequent it has occurred since leads to to believe it is the tabs at fault.

[The documentation](https://docs.steambrew.app/developers/themes/dynamic-patching) doesn't mention the tab key for the skin.json.

**Comment by shdwmtr** [comment]
If any documentation is missing I encourage anymore to contribute to it, It's alot to manage myself.

**Comment by yurisuika** [comment]
Ah, we can add documentation? I can certainly help. For me the [Contribute](https://github.com/SteamClientHomebrew/Docs/fork) link at the top of the documentation page leads to a 404 page, however. Perhaps some credentials are required?

**Comment by shdwmtr** [comment]
You're totally right, forgot to update it after switching repository names. Just updated it on the docs page.

**Comment by shdwmtr** [comment]
Fixed in latest pre-release. Full release should follow soon.
